src/main/java/uk/gov/cshr/report/reports/BookingReportRow.java

```diff
@@ -30,176 +33,8 @@
     private String bookingCancellationReason;
     private String topicId;
     private String poNumber;
-
```

Comment:
@KLV96 @kameshganesan

We actually removed lombok from this class as it was causing some issues in the past with the ordering of data on the csv. See the following commit:
https://github.com/Civil-Service-Human-Resources/lpg-report-service/commit/354d7c17f73fc4c26b648faf376783d1b468c130#diff-dc9c89b6d759619c69abb8ffed04deb4

Can you reinstate please?

Reply:
done